.github/workflows/trivy_scan_before_release.yml

```diff
@@ -0,0 +1,49 @@
+name: Scan Theia Docker image for vulnerabilities before release
+
+on:
+  pull_request:
+    branches:
+      - release-*
+
+jobs:
+  build:
+    if: startsWith(github.event.pull_request.title, 'Release ')
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image:
+          - spark-jobs
+          - clickhouse-monitor
+          - clickhouse-server
+    steps:
+    - uses: actions/checkout@v3
+    - name: Build theia Docker images
+      id: make
+      run: |
+        make ${{ matrix.image }}
+    - name: Run Trivy vulnerability scanner on latest Theia ${{ matrix.image }} image
+      if: ${{ always() && steps.make.conclusion == 'success' }}
+      uses: aquasecurity/trivy-action@0.10.0
+      continue-on-error: true
+      # we cannot use .trivy.yml as we need to override some config parameters
+      # and that is not supported by aquasecurity/trivy-action
+      with:
+        scan-type: 'image'
+        image-ref: 'antrea/theia-${{ matrix.image }}:latest'
+        trivy-config: '.trivy.yml'
+  build_theia_manager:
```

Comment:
it's not obvious to me why `theia-manager` is not just another case in the previous job's matrix?

Reply:
I did it this way as for all other images they do not have “theia-“ prefix while manager does, so either I change the Strings of all the make cases or I add an additional job here, so I preferred doing the later and the code change is quite less as compared to the former.

Reply:
Sounds good, that's what I was missing